src/hub.html

```diff
@@ -329,6 +329,7 @@
                   missOpacity: 0.1;
                   curveShootingSpeed: 12;"
               haptic-feedback
+              event-repeater="events: haptic_pulse; eventSource: #cursor"
```

Comment:
Add this to the left controller as well?

Reply:
Actually that brings up a good point. On 6-dof, having this on the left largely doesn't matter as your cursor is only ever on the right hand. However, that is not the case for 3dof devices that may be left-handed only. That said, haptics only exist on 6-dof right now, and so this should be sufficient. (If we did add it to the left anyways, both controllers would get a haptic pulse when the cursor triggered one).

Reply:
We currently only support right hand on 3dof anyway, so this sint an issue, though we obviously need to address the larger issue of handedness throughout.

Reply:
>We currently only support right hand on 3dof anyway
Is not true, we respect if you have your controller set to left-handed on go, gearvr and daydream